tests/test_model.py

```diff
@@ -1653,3 +1654,22 @@ def test_model_logp_fast_compile():
 
     with pytensor.config.change_flags(mode="FAST_COMPILE"):
         assert m.point_logps() == {"a": -1.5}
+
+
+class TestModelGraphs:
+    @staticmethod
+    def school_model() -> pm.Model:
+        J = 8
+        y = np.array([28, 8, -3, 7, -1, 1, 18, 12])
+        sigma = np.array([15, 10, 16, 11, 9, 11, 10, 18])
+        with pm.Model() as schools:
+            eta = pm.Normal("eta", 0, 1, shape=J)
+            mu = pm.Normal("mu", 0, sigma=1e6)
+            tau = pm.HalfCauchy("tau", 25)
+            theta = mu + tau * eta
+            obs = pm.Normal("obs", theta, sigma=sigma, observed=y)
+        return schools
+
+    def test_graphviz(self) -> None:
+        model = self.school_model()
+        assert model.graphviz()
```

Comment:
Can the output be compared by equality to what is returned when calling `model_graphviz(model)`?

Reply:
I tried the simple `assert pm.model_graph_model_to_graphviz(model) == model.graphviz()` but this failed because probably there is no default how to compare the output graphs. I will investigate.

Reply:
Addressed in https://github.com/pymc-devs/pymc/pull/6865/commits/3c252d02520949546b5ef4793a258cc9a2fb54ff